docs/2_league-info.md

```diff
@@ -35,7 +35,7 @@
 
 &emsp;
 
-## 2.4 CS:GO Bans
+## 2.4 Account Bans
```

Comment:
I feel like "Account Bans" is a tiny bit vague, would it be better to specify "CS/VAC/FACEIT Bans" or something like that?